### ISSUE TITLE
Rename `Ferumbras Mortal Shell` → `Ascending Ferumbras`?

### ISSUE THREAD

**Issue by mathiasbynens** [tibiamaps/tibia-kill-stats#6]
[Today's update](https://www.tibia.com/news/?subtopic=newsarchive&id=7171) renamed at least one boss in the client Bosstiary: `Ferumbras Mortal Shell` → `Ascending Ferumbras`. If this change affects the kill stats as well, we need to [merge the old entries](https://github.com/tibiamaps/tibia-kill-stats/blob/main/merge-entries.mjs) after waiting 7 days first. If not, we might need to simply update the normalization mapping.

**Comment by LeeKunTibia** [comment]
Yes, as anticipated the boss does show up as `Ascending Ferumbras` on kill statistics now. Similar to Shlorg and Tentugly, changing its name for the Bosstiary also changed how it appears on kill statistics.

**Comment by mathiasbynens** [comment]
What were the old names for Shlorg and Tentugly? I might need to retroactively correct this repo's data.

**Comment by LeeKunTibia** [comment]
`Weakened Shlorg` and `Tentugly's Head`, but they were changed with the Summer Update in 2022 (July 18th), before the data collection started here as far as I can tell.

**Comment by mathiasbynens** [comment]
> `Weakened Shlorg` and `Tentugly's Head`, but they were changed with the Summer Update in 2022 (July 18th), before the data collection started here as far as I can tell.

I see. Indeed, nothing to do here then.

> Yes, as anticipated the boss does show up as `Ascending Ferumbras` on kill statistics now.

Note that `Ascending Ferumbras` already showed up in kill stats previously, so the fact it apears in kill stats doesn't necessarily mean anything changed.

The latest kill stats update still increased the counter for `Ferumbras Mortal Shell`: https://github.com/tibiamaps/tibia-kill-stats/commit/dfe9188b82ffb76bfbb7cdde56de8e88b951e35e#diff-6b45b3bc7156647a35db592527979048f37a3e01bca294ca2fe23997410100c1 So it could be that this continues to be the name to look for in kill stats (if the goal is to count the number of completed boss kills, at least).

**Comment by LeeKunTibia** [comment]
> Note that `Ascending Ferumbras` already showed up in kill stats previously, so the fact it apears in kill stats doesn't necessarily mean anything changed.

But that used to be only when players died to that stage of the boss, which could happen but was very unusual. Since yesterday's update, however, it started appearing with the boss being killed - for the first time since 2016. 

> The latest kill stats update still increased the counter for `Ferumbras Mortal Shell`

That's only because of the lack of sync between the kill-statistics and the server save. These last additions to Mortal Shell were killed between the previous website update and the server save which implemented the change. If I am right, then we shouldn't see any more additions starting with the next update after this post. 

The change in pattern can easily be seen when looked at the aggregated data for `Ascending Ferumbras` on GuildStats:

https://guildstats.eu/bosses?world=&monsterName=Ascending%20Ferumbras&pageno=1
